src/ReflectionEngine.php

```diff
@@ -80,6 +81,7 @@ public static function init(LocatorInterface $locator): void
         self::$traverser = $traverser = new NodeTraverser();
         $traverser->addVisitor(new NameResolver());
         $traverser->addVisitor(new RootNamespaceNormalizer());
+        $traverser->addVisitor(new ParentConnectingVisitor());
```

Comment:
this is needed to go up from `PropertyProperty` to get `Property`, which has `attrGroups`, via `parent` attribute.